demisto_sdk/commands/validate/validators/DO_validators/DO102_docker_image_is_latest_tag.py

```diff
@@ -0,0 +1,121 @@
+    )
+    fix_message = "docker image {0} has been updated to {1}"
+    related_field = "Docker image"
+    is_auto_fixable = True
+
+    def is_docker_image_older_than_three_days(self, docker_image) -> bool:
+        """
+        Return True if the docker image is more than 3 days old.
+
+        Returns:
+            bool: True if the docker is more than 3 days old.
+        """
+        three_days_ago: datetime = parse("3 days ago")  # type: ignore[assignment]
+        last_updated = self.dockerhub_client.get_docker_image_tag_creation_date(
+            docker_image.name, tag=docker_image.tag
+        )
+        return not last_updated or three_days_ago > last_updated
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        invalid_content_items = []
+        for content_item in content_items:
+            if not content_item.is_javascript:
+                docker_image = content_item.docker_image_object
+                if not docker_image.is_valid:
+                    invalid_content_items.append(
+                        ValidationResult(
+                            validator=self,
+                            message=f"Docker image {content_item.docker_image} format is invalid, cannot determine if it uses the latest tag",
+                            content_object=content_item,
+                        )
+                    )
+                    continue
+                try:
+                    docker_image_latest_tag = str(
+                        self.dockerhub_client.get_latest_docker_image_tag(
+                            docker_image.name
+                        )
+                    )
+                except DockerHubRequestException as error:
+                    invalid_content_items.append(
+                        ValidationResult(
+                            validator=self,
+                            message=str(error),
+                            content_object=content_item,
+                        )
+                    )
+                    continue
+                if (
+                    docker_image.tag != docker_image_latest_tag
+                    and self.is_docker_image_older_than_three_days(docker_image)
+                ):
+                    invalid_content_items.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message.format(
+                                content_item.docker_image,
+                                docker_image.tag,
+                                docker_image_latest_tag,
+                            ),
+                            content_object=content_item,
+                        )
+                    )
+        return invalid_content_items
+
+    def fix(
+        self,
+        content_item: ContentTypes,
+    ) -> FixResult:
+        docker_image = content_item.docker_image_object
+        try:
+            latest_tag = str(
```

Comment:
Why not working with a dict to keep the latest tag for each docker type rather than making the call each time?

Reply:
i already have a cache in `DockerHubClient, hence i didn't implement it.